cpp/src/arrow/c/bridge_test.cc

```diff
@@ -1004,15 +1006,16 @@ TEST_F(TestArrayExport, List) {
 TEST_F(TestArrayExport, ListSliced) {
   {
     auto factory = []() {
-      return ArrayFromJSON(list(int8()), "[[1, 2], [3, null], [4, 5, 6], null]")
+      return arrow::ArrayFromJSON(list(int8()), "[[1, 2], [3, null], [4, 5, 6], null]")
```

Comment:
Are these necessary? It seems better to include a `using` declaration at the top of the file

Reply:
They're not. I'll scan over the files I changed and swap in using statements.

Reply:
Changed in 09c6e827469715353884a288ad1ebf741c91b70f. I think that's the best way to do that?

Reply:
Hrm, not totally sure what's best here. In this PR, I included `from_json.h` in `api.h` which has a few consequences related to your comment here. I don't think we strictly need to put things in api.h so I'm going to back that out and clean things up.

Reply:
Adding a `using` would be nicer here, e.g.:
```cpp
using arrow::ArrayFromJSON;
```